redisraft.h

```diff
@@ -244,19 +257,63 @@ typedef struct RaftMeta {
     raft_node_id_t vote;                        /* Our vote in the last term, or -1 */
 } RaftMeta;
 
+/* threadpool.c */
+struct Task {
+    STAILQ_ENTRY(Task) entry;
+    void *arg;
+    void (*run)(void *arg);
+};
+
+typedef struct ThreadPool {
+    int thread_count;
+    pthread_t *threads;
+
+    STAILQ_HEAD(tasks, Task) tasks;
+
+    pthread_mutex_t mtx;
+    pthread_cond_t cond;
+    int shutdown;
```

Comment:
Consider `bool`, this is used elsewhere.

Reply:
done